### PR TITLE
Tl ja email

### PR DESCRIPTION
**Pull request by Timothyl**
started test for email
created association for food_truck and user

### CODE REVIEW

File: spec/features/review_tests/owner_emailed_for_new_review_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  I want to receive an email when my truck is reviewed
+  So that I know what users think of it
+} do
+  # Acceptance Criteria
+  # If a food truck is reviewed, the creator of the food truck
+  # will be emailed.
+
+  scenario "food truck entry owner is emailed when new review is posted" do
+    owner = FactoryGirl.create(:user)
+    truck = FactoryGirl.create(:food_truck)
```
Comment (mlg-) [inline spec/features/review_tests/owner_emailed_for_new_review_spec.rb]: You should be able to do something like `truck = FactoryGirl.create(:food_truck, user: owner)` so you can get rid of line 15.

File: spec/features/review_tests/owner_emailed_for_new_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    click_button 'Log in'
+
+    visit food_truck_path(truck)
+
+    expect(page).to have_content("Submit Your Review")
+
+    click_button('Submit Your Review')
+
+    expect(page).to have_content("Header")
```
Comment (mlg-) [inline spec/features/review_tests/owner_emailed_for_new_review_spec.rb]: Pretty sure you already have tests that cover whether or not there's a form here, so maybe these expect statements aren't necessary for what this particular test wants to check for.

File: spec/features/review_tests/owner_emailed_for_new_review_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    expect(page).to have_content("Body")
+    expect(page).to have_content("Rating")
+
+    fill_in "Header", with: 'blahblahblah'
+    fill_in "Body", with: 'such food much good
+    such food much good such food much
+    good such food much good such food much good'
+    fill_in "Rating", with: '1'
+    click_button('Submit Review')
```
Comment (mlg-) [inline spec/features/review_tests/owner_emailed_for_new_review_spec.rb]: Hmm, how exactly do we know anything about what the test purports to discover at this point? Could you check some attributes of `ActionMailer::Base.deliveries`?

Comment (Timothyl) [inline spec/features/review_tests/owner_emailed_for_new_review_spec.rb]: We haven't figured out ActionMailer yet, so we're not quite finished with this test yet. We'll be coming back to it.

Comment (mlg-) [inline spec/features/review_tests/owner_emailed_for_new_review_spec.rb]: Okay, fair enough!

**Pull request merged**